sdks/python/setup.py

```diff
@@ -175,7 +175,8 @@ def get_version():
     'pytest-timeout>=1.3.3,<2',
     'sqlalchemy>=1.3,<2.0',
     'psycopg2-binary>=2.8.5,<3.0.0',
-    'testcontainers>=3.0.3,<4.0.0',
+    'testcontainers[mysql]>=3.0.3,<4.0.0',
+    'cryptography>=36.0.0',
```

Comment:
Why was this needed ?

Reply:
it's needed to connect to the MySQL container, interestingly : )